### PR TITLE
fix(OverflowMenu): pass HTMLDivElements props to group and control divs

### LINKED ISSUE #6478: OverflowMenuGroup/OverflowMenuControl do not forward props
I should be able to add other valid HTML props to OverflowMenuGroup/OverflowMenuControl

### PR DESCRIPTION
**Pull request by boaz0**
**What**: Closes #6478 

**Additional issues**: None

### CODE REVIEW

File: packages/react-core/src/components/OverflowMenu/OverflowMenuControl.tsx [hunk 1]
```diff
@@ -15,12 +15,16 @@ export interface OverflowMenuControlProps extends React.HTMLProps<HTMLDivElement
...
   children,
-  hasAdditionalOptions
+  hasAdditionalOptions,
+  ...props
 }: OverflowMenuControlProps) => (
   <OverflowMenuContext.Consumer>
     {value =>
       (value.isBelowBreakpoint || hasAdditionalOptions) && (
-        <div className={css(styles.overflowMenuControl, className)}> {children} </div>
+        <div {...props} className={css(styles.overflowMenuControl, className)}>
```
Comment (tlabaj) [inline packages/react-core/src/components/OverflowMenu/OverflowMenuControl.tsx:24]: Can you please spread props at the end.

File: packages/react-core/src/components/OverflowMenu/OverflowMenuGroup.tsx [hunk 2]
```diff
@@ -18,12 +18,14 @@ export const OverflowMenuGroup: React.FC<OverflowMenuGroupProps> = ({
...
   isPersistent = false,
-  groupType
+  groupType,
+  ...props
 }: OverflowMenuGroupProps) => (
   <OverflowMenuContext.Consumer>
     {value =>
       (isPersistent || !value.isBelowBreakpoint) && (
         <div
+          {...props}
```
Comment (tlabaj) [inline packages/react-core/src/components/OverflowMenu/OverflowMenuGroup.tsx:28]: same comment here.

File: packages/react-core/src/components/OverflowMenu/OverflowMenuControl.tsx (hunk 1 above)
Comment (boaz0) [reply packages/react-core/src/components/OverflowMenu/OverflowMenuControl.tsx]: Done 👍

File: packages/react-core/src/components/OverflowMenu/OverflowMenuGroup.tsx (hunk 2 above)
Comment (boaz0) [reply packages/react-core/src/components/OverflowMenu/OverflowMenuGroup.tsx]: Done 👍

**Review by nicolethoen — APPROVED** [review]

**Review by tlabaj — APPROVED** [review]

**Pull request merged**